cmd/snap/cmd_interfaces.go

```diff
@@ -100,19 +121,20 @@ func (x *cmdInterfaces) Execute(args []string) error {
 		}
 		// The OS snap is special and enable abbreviated
 		// display syntax on the slot-side of the connection.
-		if slot.Snap == "core" || slot.Snap == "ubuntu-core" {
+		if slot.Snap == "core" || slot.Snap == "ubuntu-core" || slot.Snap == snap.Nickname("core") {
 			fmt.Fprintf(w, ":%s\t", slot.Name)
 		} else {
-			fmt.Fprintf(w, "%s:%s\t", slot.Snap, slot.Name)
+			fmt.Fprintf(w, "%s:%s\t", maybeAsNickname(slot.Snap, askedWithNick), slot.Name)
```

Comment:
Doesn't feel like we gain much from varying here based on the snap. Just feels more magic, which isn't necessarily good. In fact, the case for core, which is the system snap and the only nickname we care about right now, is already special case here in a custom branch right above. So the change seems to introduce extra complexity without much of a win. Plus, this whole command is a bit unfortunate.. we really need that "connections" command we discussed a few times.

So perhaps we should keep this simpler and more straightforward for now.

Reply:
Are you suggesting I should drop `maybeAsNickname()` here, as well as where connections and unconnected plugs are printed or just this particular location?

Reply:
We discussed this on IRC